Abdullah_AlQahtani/week_01/day_02/Variables_LAB/main.js

```diff
@@ -0,0 +1,29 @@
+//The Fortune Teller
+console.log(" == The Fortune Teller ==");
+var numOfChild = 5;
+var partner = "Sara";
+var geographic = "Dammam";
+var job = "Manager of IT department";
+
+console.log("You will be a " + job + "in " + geographic + ", and married to " + partner + "with " + numOfChild + "kids.");
```

Comment:
Good work, when printing long sentences it's a good idea to make sure to have enough spaces between words to make them more readable.